lib/manageiq/providers/amazon/engine.rb

```diff
@@ -1,4 +1,5 @@
-require 'rails/all'
+require 'rails/engine'
+require 'action_controller/railtie' # needed for isolate_namespace below
```

Comment:
ah, sounds like a dependency race condition is lurking there...

as said, I dont think we need to require anything here, thats how we started off at least